src/entity/Living.php

```diff
@@ -326,6 +327,9 @@ public function fall(float $fallDistance) : void{
 				$this->broadcastSound(new EntityLandSound($this, $fallBlock));
 			}
 		}
+		if($fallDistance > 0.99 && $fallBlock->getId() === BlockLegacyIds::FARMLAND) {
+			$this->getWorld()->setBlock($fallBlockPos, BlockFactory::getInstance()->get(BlockLegacyIds::DIRT, 0));
```

Comment:
```php
VanillaBlocks::DIRT()
```
would be better on here

Reply:
There's no guarantee this will actually work at all, since farmland has a 1x1x1 collision box. This means that the fall position collected above will actually be the block above the farmland in every case (unless a rounding error occurs).